pf/internal/check/check_test.go

```diff
@@ -96,6 +155,97 @@ func TestSensitiveIDWithOverride(t *testing.T) {
 	})
 }
 
+func TestInvalidInputID(t *testing.T) {
+	tests := []struct {
+		name            string
+		idSchema        idSchema
+		isInputProperty bool
```

Comment:
Superficially reading this, I think isInputProperty can be dropped from the test specification and instead computed by `isInputProperty`, this would make the test a tad more focused.

Reply:
Done!